src/Events/Telemetry/Telemetry.php

```diff
@@ -104,20 +104,24 @@ public function filter_tribe_general_settings_debugging_section( $fields ): arra
 			'type'            => 'checkbox_bool',
 			'label'           => $label,
 			'tooltip'         => sprintf(
-				/* Translators: Description of the Telemetry optin setting.
-				%1$s: opening anchor tag for permissions link.
-				%2$s: opening anchor tag for terms of service link.
-				%3$s: opening anchor tag for privacy policy link.
-				%4$s: closing anchor tags.
-				*/
+			/* Translators: Description of the Telemetry optin setting.
+			%1$s: opening anchor tag for permissions link.
+			%2$s: opening anchor tag for terms of service link.
+			%3$s: opening anchor tag for privacy policy link.
+			%4$s: closing anchor tags.
+			*/
 				_x(
-					'Enable this option to share usage data with The Events Calendar and StellarWP. %1$sWhat permissions are being granted?%4$s %2$sRead our terms of service%4$s. %3$sRead our privacy policy%4$s.',
-					'Description of optin setting.',
+					'Enable this option to share usage data with The Events Calendar and StellarWP.
+        This activates access to TEC AI chatbot and in-app priority support for premium users.
+        <br/>%1$sWhat permissions are being granted?%4$s
+        <br/>%2$sRead our terms of service%4$s.
+        <br/>%3$sRead our privacy policy%4$s.',
+					'Description of opt-in setting.',
 					'the-events-calendar'
 				),
-				'<a href=" ' . Common_Telemetry::get_permissions_url() . ' ">',
-				'<a href=" ' . Common_Telemetry::get_terms_url() . ' ">',
-				'<a href=" ' . Common_Telemetry::get_privacy_url() . ' ">',
+				'<a href="' . Common_Telemetry::get_permissions_url() . '">',
+				'<a href="' . Common_Telemetry::get_terms_url() . '">',
+				'<a href="' . Common_Telemetry::get_privacy_url() . '">',
```

Comment:
esc_url?

Reply:
Or, per Slack convo: drop a comment in here so a reviewer knows it's escaped in the function.